qiskit/optimization/problems/quadratic_program.py

```diff
@@ -1092,6 +1093,63 @@ def from_ising(self,
         self.minimize(constant=offset, linear=linear_terms, quadratic=quadratic_terms)
         offset -= offset
 
+    def is_feasible(self, x: Union[List[float], np.ndarray], detailed: bool = False) -> bool:
+        """Returns whether a solution provided by the optimizer is feasible or not
```

Comment:
I am sorry, I didn't get the 2nd part of your suggestion.

Reply:
I meant that however you come up with a solution to the problem that the method `Returns whether a solution is feasible or not` and is in no way dependent on a solution as output from one of the optimizers. It was not meant to be 2 parts, more an explanation of why I thought the shorter version was more applicable.

Reply:
Correct, I saw the idea, however I thought that we are implementing this in the optimizers and it would hardly be used directly( like problem.is_feasible(custom_input) ), I will change it to your suggestion as it looks more dynamic.